doc/api/next_api_changes/2018-05-26-TH.rst

```diff
@@ -0,0 +1,9 @@
+Removed proprietary sphinx directives
```

Comment:
I don't know if I'd call these proprietary; the word that I'd like to suggest escapes me at the moment, though.

Reply:
"internally defined"? I agree that I wouldn't call them proprietary.

Reply:
“In-house”?